vnext/Microsoft.ReactNative/ReactNative.V8JSI.Windows.WinUI3.targets

```diff
@@ -0,0 +1,21 @@
+<Project ToolsVersion="4.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <ItemDefinitionGroup>
+    <Link>
+      <AdditionalLibraryDirectories Condition="'$(Configuration)' == 'WinUI3' And '$(Platform)' == 'x64'">$(V8_Package)\build\native\..\..\lib\Debug\x64;%(AdditionalLibraryDirectories)</AdditionalLibraryDirectories>
+      <AdditionalLibraryDirectories Condition="'$(Configuration)' == 'WinUI3' And ('$(Platform)' == 'Win32' Or '$(Platform)' == 'x86')">$(V8_Package)\build\native\..\..\lib\Debug\x86;%(AdditionalLibraryDirectories)</AdditionalLibraryDirectories>
+      <AdditionalDependencies>v8jsi.dll.lib;%(AdditionalDependencies)</AdditionalDependencies>
+      <DelayLoadDLLs>v8jsi.dll;%(DelayLoadDLLs)</DelayLoadDLLs>
+    </Link>
+    <ClCompile>
+      <AdditionalIncludeDirectories>$(V8_Package)\build\native\include;%(AdditionalIncludeDirectories)</AdditionalIncludeDirectories>
+    </ClCompile>
+  </ItemDefinitionGroup>
+  <ItemGroup Condition="'$(Configuration)' == 'WinUI3' And '$(Platform)' == 'x64' And '$(V8JSI_NODLLCOPY)' == ''">
```

Comment:
How is this different from the normal V8JSI targets file?

Reply:
the build outputs go to \vnext\target\$(Configuration) - adding a new configuration means that the build will expect certain artifacts like DLLs/LIBs to be available in that target configuration path. Normally nuget packages include props/targets for Debug/Release, since we're adding a new configuration we have to copy this for the new custom configuration.